integration/node_start_test.go

```diff
@@ -133,5 +136,87 @@ func testNodeStart(t *testing.T, context spec.G, it spec.S) {
 				Expect(string(content)).To(ContainSubstring("hello world"))
 			})
 		})
+
+		context("when using CA certificates", func() {
+			var (
+				client *http.Client
+			)
+
+			it.Before(func() {
+				var err error
+				name, err = occam.RandomName()
+				Expect(err).NotTo(HaveOccurred())
+				source, err = occam.Source(filepath.Join("testdata", "ca_certs"))
+				Expect(err).NotTo(HaveOccurred())
+
+				caCert, err := ioutil.ReadFile(fmt.Sprintf("%s/client/ca.pem", source))
+				Expect(err).ToNot(HaveOccurred())
+
+				caCertPool := x509.NewCertPool()
+				caCertPool.AppendCertsFromPEM(caCert)
+
+				cert, err := tls.LoadX509KeyPair(fmt.Sprintf("%s/client/cert.pem", source), fmt.Sprintf("%s/client/key.pem", source))
+				Expect(err).ToNot(HaveOccurred())
+
+				client = &http.Client{
+					Transport: &http.Transport{
+						TLSClientConfig: &tls.Config{
+							RootCAs:      caCertPool,
+							Certificates: []tls.Certificate{cert},
+							MinVersion:   tls.VersionTLS12,
+						},
+					},
+				}
+			})
+
+			it("builds a working OCI image and uses a client-side CA cert for requests", func() {
+				var err error
+				var logs fmt.Stringer
+				image, logs, err = pack.WithNoColor().Build.
+					WithBuildpacks(nodeBuildpack).
+					WithPullPolicy("never").
+					Execute(name, filepath.Join(source, "server"))
+				Expect(err).NotTo(HaveOccurred())
+
+				Expect(logs).To(ContainLines(ContainSubstring("CA Certificates Buildpack")))
+				Expect(logs).To(ContainLines(ContainSubstring("Node Engine Buildpack")))
+				Expect(logs).To(ContainLines(ContainSubstring("Node Start Buildpack")))
+
+				container, err = docker.Container.Run.
+					WithPublish("8080").
+					WithEnv(map[string]string{
+						"PORT":                 "8080",
+						"SERVICE_BINDING_ROOT": "/bindings",
+						"NODE_OPTIONS":         "--use-openssl-ca",
```

Comment:
`NODE_OPTIONS` is not required since the node binary is compiled with `--openssl-use-def-ca-store`. See https://github.com/paketo-buildpacks/nodejs/pull/341/commits/4013e6a2fe2cfd0a247e0cc7788ead1feba020a0 for updates.